### PR TITLE
feat: Execute aggregate query and finish steps

### LINKED ISSUE #3893: firstMile: Wizard Step 7: Execute an Aggregate
User Story: Step 7: Execute an Aggregate

    ACs

    - When the user is on the Execute an Aggregate step, highlight that portion of the wizard
    - Display the text from the Figma Prototype
    - The user can use a copy code component to copy/paste code snippets as written in the Figma prototype
    -  When the user clicks the "Next" button, navigate the user to the next step in the wizard and checkmark the overview step as being completed (blue checkmark)
    - When the user clicks the "PREVIOUS" button, navigate the user to the previous step in the wizard

### LINKED ISSUE #3894: firstMile: Wizard Step 8: Finish Elements
User Story: Step 8: Finish Elements & Text

ACs

    - When the user is on the Finish step, highlight that portion of the wizard
    - Display the text from the Figma Prototype; i.e., "Congrats" ...
    - The screen displays elements according to Figma with Icon and Text.
    - When the user clicks on the element, the system will have the ability to do one of the following, depending on defined requirements (link to a specific URL or link to a defined URL within the application)

   Dev Notes
   - Create these components in Clockface rather than UI

### PR DESCRIPTION
**Pull request by ChitlangeSahas**
Closes #3893 
Closes #3894 

Screen recording:

https://user-images.githubusercontent.com/18511823/157739872-5eb5e364-f1d4-49c0-a1d5-5f0d60e9794b.mov

### CODE REVIEW

File: src/homepageExperience/containers/HomepagePythonWizard.tsx [hunk 1]
```diff
@@ -55,6 +57,12 @@ export class HomepagePythonWizard extends PureComponent<null, State> {
       case 6: {
         return <ExecuteQuery />
       }
+      case 7: {
+        return <ExecuteAggregateQuery />
+      }
+      case 8: {
+        return <Finish />
+      }
```
Comment (hoorayimhelping) [inline src/homepageExperience/containers/HomepagePythonWizard.tsx:65]: along with this, let's add a check for the `Next` button that if we're on step 8, don't render it. We should probably do the same thing with step 1 and the `Previous` button.

File: src/homepageExperience/components/HomepageExperience.scss [hunk 2]
```diff
@@ -156,3 +159,39 @@ $pool-fill: #00a3ff;
   align-items: center;
   text-transform: uppercase;
 }
+
+.homepage-wizard-next-steps {
```
Comment (hoorayimhelping) [inline src/homepageExperience/components/HomepageExperience.scss:163]: in a follow up ticket (not this one), can you style up the `HomePageContainer` similarly to the way this page is styled?

**Review by hoorayimhelping — APPROVED** [review]
looks great!

File: src/homepageExperience/containers/HomepagePythonWizard.tsx (hunk 1 above)
Comment (ChitlangeSahas) [reply src/homepageExperience/containers/HomepagePythonWizard.tsx:70]: Done!

Comment (ChitlangeSahas) [reply src/homepageExperience/containers/HomepagePythonWizard.tsx:70]: I used the `status` prop of out Button component.

**Pull request merged**